### PR TITLE
fix: broken url on booking page

### LINKED ISSUE #15021: Back to Homapage flow not working
Found a bug? Please fill out the sections below. 👍

### Issue Summary

On clicking the profile icon, the site doesn't go back to the profile homepage where all the meetings are listed. Earlier it used to.

### Steps to Reproduce

1. Went to https://cal.com/urbanfarms
2. Clicked on a meeting. Example: [https://cal.com/urbanfarms/movie-under-the-stars?date=2024-05-24&month=2024-05](url)
3. Clicked on profile icon on top-left
Screenshot 2024-05-14 at 11 55 23 AM
4. The site went to a 404 page
Screenshot 2024-05-14 at 10 40 03 AM

### Actual Results

- Clicking on profile icon is taking to a 404 page

### Expected Results

- Clicking on profile icon should take to profile homepage

### Technical details

- Browser: Safari version 16.6 | Chrome Version 124.0.6367.201

### Evidence

attaching screen recording

https://github.com/calcom/cal.com/assets/169751971/fa4799ef-ae19-4ad7-bbcd-c858e371d9a9

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

- Fixes https://github.com/calcom/cal.com/issues/15021
- Fixes CAL-3702 (Linear issue number - should be visible at the bottom of the GitHub issue description)

Regression from https://github.com/calcom/cal.com/pull/14777/files.

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected)
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com)
- [ ] I have added or modified automated tests that prove my fix is effective or that my feature works (PRs might be rejected if logical changes are not properly tested)

## How should this be tested?

1) Go to any team event type url and make sure team logo is not visible. (for non org event types)

### CODE REVIEW

File: packages/features/bookings/components/event-meta/Members.tsx [hunk 1]
```diff
@@ -28,19 +28,25 @@ export const EventMembers = ({ schedulingType, users, profile, entity }: EventMe
...
+  const orgAvatarItem = entity.orgSlug
+    ? [
+        {
+          // We don't want booker to be able to see the list of other users or teams inside the embed
+          href: isEmbed ? null : getBookerBaseUrlSync(entity.orgSlug),
+          image: profile.image || "",
+          alt: profile.name || "",
+          title: profile.name || "",
+        },
+      ]
```
Comment (Udit-takkar) [inline packages/features/bookings/components/event-meta/Members.tsx:40]: This was only meant for org event types and not normal event types

**Review by zomars — APPROVED** [review]

**Review by zomars — DISMISSED** [review]
We're having merge conflicts now. @Udit-takkar Can you take a look?

File: packages/features/bookings/components/event-meta/Members.tsx [hunk 2]
```diff
@@ -30,23 +30,26 @@ export const EventMembers = ({ schedulingType, users, profile, entity }: EventMe
...
-                {
-                  // We don't want booker to be able to see the list of other users or teams inside the embed
-                  href: isEmbed ? null : getBookerBaseUrlSync(entity.orgSlug),
-                  image: profile.image || "",
-                  alt: profile.name || "",
-                  title: profile.name || "",
-                },
-              ]),
+          ...orgAvatarItem,
           ...shownUsers.map((user) => ({
```
Comment (hariombalhara) [inline packages/features/bookings/components/event-meta/Members.tsx:53]: `shownUsers` is empty array for a non-org user event. That should also be fixed. We should show the user avatar for such an event and that should take to the user profile page.

This is what the reporter also mentioned here.
image

**Review by hariombalhara — DISMISSED** [review]
Left a blocking comment otherwise it is good.

**Review by hariombalhara — APPROVED** [review]

**Comment by hariombalhara** [comment]
Approved but let's add a test for this in a follow-up

**Pull request merged**